### DISCUSSION TITLE
Services disappeared from the control panel but continue to run.

### DISCUSSION THREAD

**Discussion by vitaliyhan** [Dokploy/dokploy#806] [Q&A]
I have projects - sites, eccex, eccex-dev that are missing services, but yesterday everything was fine.
2 services from sites, eccex and eccex-dev project, and one db from db project. 
<img width="698" alt="image" src="https://github.com/user-attachments/assets/3225a117-cc25-4f17-aff8-a9bf6125c84a">
But i can see them running in Docker tab and i can reach them, but cant controll them using dashboard. 
<img width="758" alt="image" src="https://github.com/user-attachments/assets/607f065c-97dc-434e-93d4-8f9d95cfe936">

What can i do to make them show up in dashboard again, without reinstalling everything, like i did 2 times previously.
And how to prevent this behavior. 
Thanks.

**Comment by Lichhu** [comment]
What version are you currently on?

**Reply by vitaliyhan** [reply]
I am on version: v0.11.2, also i experianced the same on v0.9.4 and earlier. 
P.S. I updated to last version, no changes
i have 4 vCPU / 8 GB RAM,80 GB SSD, Ubuntu 22.04 LTS server
i have a lot of free space (used 28 from 80)
All worked 19 days with no problem. All i did, only autodeploy services and setup database mysql service (using "+create service" -> database-> mysql8, that i can see fine in runnig docker)
I have Daily Docker Cleanup turned on.

Also just find out that my remote server settings disappear, but i didnt use that server to deploy so thats ok.  

What kind of log i can provide to make situation more clear? Thanks.

**Comment by vitaliyhan** [comment]
i moved to coolify.
